File: samples/server/petstore/jaxrs-cxf-cdi/src/main/java/io/swagger/api/impl/UserApiServiceImpl.java
package io.swagger.api.impl;

import io.swagger.api.*;
import io.swagger.model.*;

import org.apache.cxf.jaxrs.ext.multipart.Attachment;

import java.util.List;
import io.swagger.model.User;

import java.util.List;

import java.io.InputStream;

import javax.enterprise.context.RequestScoped;
import javax.ws.rs.core.Response;
import javax.ws.rs.core.SecurityContext;

@RequestScoped

public class UserApiServiceImpl implements UserApiService {
      @Override
      public Response createUser(User body, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response createUsersWithArrayInput(List<User> body, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response createUsersWithListInput(List<User> body, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response deleteUser(String username, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response getUserByName(String username, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response loginUser(String username, String password, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response logoutUser(SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response updateUser(String username, User body, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
}

File: samples/server/petstore/kotlin-server/javalin-6/src/main/kotlin/org/openapitools/server/apis/UserApiServiceImpl.kt
package org.openapitools.server.apis

import java.time.OffsetDateTime
import org.openapitools.server.models.User
import io.javalin.http.Context

class UserApiServiceImpl : UserApiService {

    override fun createUser(user: User, ctx: Context): Unit {
        TODO("Implement me")
    }

    override fun createUsersWithArrayInput(user: List<User>, ctx: Context): Unit {
        TODO("Implement me")
    }

    override fun createUsersWithListInput(user: List<User>, ctx: Context): Unit {
        TODO("Implement me")
    }

    override fun deleteUser(username: String, ctx: Context): Unit {
        TODO("Implement me")
    }

    override fun getUserByName(username: String, ctx: Context): User {
        TODO("Implement me")
    }

    override fun loginUser(username: String, password: String, ctx: Context): String {
        TODO("Implement me")
    }

    override fun logoutUser(ctx: Context): Unit {
        TODO("Implement me")
    }

    override fun updateUser(username: String, user: User, ctx: Context): Unit {
        TODO("Implement me")
    }
}
